tdrs-backend/tdpservice/settings/common.py

```diff
@@ -329,3 +305,44 @@ class Common(Configuration):
 
     # The number of seconds to wait for socket response from clamav-rest
     AV_SCAN_TIMEOUT = os.getenv('AV_SCAN_TIMEOUT', 30)
+
+    ###
+    # Authentication Provider Settings
+    #
+    LOGIN_GOV_ACR_VALUES = os.getenv(
+        'ACR_VALUES',
+        'http://idmanagement.gov/ns/assurance/ial/1'
+    )
+    LOGIN_GOV_AUTHORIZATION_ENDPOINT = os.getenv(
+        'OIDC_OP_AUTHORIZATION_ENDPOINT',
+        'https://idp.int.identitysandbox.gov/openid_connect/authorize'
+    )
+    LOGIN_GOV_CLIENT_ASSERTION_TYPE = os.getenv(
+        'CLIENT_ASSERTION_TYPE',
+        'urn:ietf:params:oauth:client-assertion-type:jwt-bearer'
+    )
+    LOGIN_GOV_CLIENT_ID = os.getenv(
+        'OIDC_RP_CLIENT_ID',
+        'urn:gov:gsa:openidconnect.profiles:sp:sso:hhs:tanf-proto-dev'
+    )
+    LOGIN_GOV_ISSUER = os.getenv(
+        'OIDC_OP_ISSUER',
+        'https://idp.int.identitysandbox.gov/'
+    )
+    LOGIN_GOV_JWKS_ENDPOINT = os.getenv(
+        'OIDC_OP_JWKS_ENDPOINT',
+        'https://idp.int.identitysandbox.gov/api/openid_connect/certs'
+    )
+    # JWT_KEY must be set, there is no functional default.
+    LOGIN_GOV_JWT_KEY = get_required_env_var_setting(
```

Comment:
`JWT_KEY` is a true secret, so it must be defined where needed and preferably nowhere else.